osaca/parser/parser_AArch64.py

```diff
@@ -571,35 +608,44 @@ def parse_register(self, register_string):
 
     def is_gpr(self, register):
         """Check if register is a general purpose register"""
-        if register["prefix"] in "wx":
+        if register.prefix in "wx":
             return True
         return False
 
     def is_vector_register(self, register):
         """Check if register is a vector register"""
-        if register["prefix"] in "bhsdqvz":
+        if register.prefix in "bhsdqvz":
             return True
         return False
 
     def is_flag_dependend_of(self, flag_a, flag_b):
         """Check if ``flag_a`` is dependent on ``flag_b``"""
         # we assume flags are independent of each other, e.g., CF can be read while ZF gets written
         # TODO validate this assumption
-        if flag_a["name"] == flag_b["name"]:
+        if isinstance(flag_a, Operand):
```

Comment:
See comments to this function in `parser_x86att.py`

Reply:
https://github.com/RRZE-HPC/OSACA/commit/fa95293cb013ca502f231bc60991f5c419b650f4